src/Views/Form/Templates/Sequoia/Sequoia.php

```diff
@@ -279,6 +279,13 @@ public function loadScripts() {
 
 		wp_enqueue_script( 'give-sequoia-template-js', GIVE_PLUGIN_URL . 'assets/dist/js/give-sequoia-template.js', [ 'give' ], GIVE_VERSION, true );
 		wp_localize_script( 'give-sequoia-template-js', 'sequoiaTemplateOptions', $templateOptions );
+		wp_localize_script(
+			'give-sequoia-template-js',
+			'sequoiaTemplateL10n',
```

Comment:
Whenever we bump minimum version to WP 5.0+ for Give we'll be able to swap this logic for the wp.i18n package, but until then this is a good workaround!